backend/src/main/kotlin/app/ehrenamtskarte/backend/matomo/Matomo.kt

```diff
@@ -0,0 +1,159 @@
+package app.ehrenamtskarte.backend.matomo
+
+import app.ehrenamtskarte.backend.config.MatomoConfig
+import app.ehrenamtskarte.backend.config.ProjectConfig
+import app.ehrenamtskarte.backend.stores.webservice.schema.SearchParams
+import app.ehrenamtskarte.backend.verification.database.CodeType
+import app.ehrenamtskarte.backend.verification.database.repos.CardRepository
+import app.ehrenamtskarte.backend.verification.webservice.schema.types.CardGenerationModel
+import jakarta.servlet.http.HttpServletRequest
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.matomo.java.tracking.MatomoRequest
+import org.matomo.java.tracking.MatomoRequestBuilder
+import org.matomo.java.tracking.MatomoTracker
+import org.slf4j.LoggerFactory
+import java.io.IOException
+import java.util.concurrent.ExecutionException
+
+enum class EventCategory {
+    Cards
+}
+
+object Matomo {
+    val logger = LoggerFactory.getLogger(Matomo::class.java)
+    private const val ACTION_NAME = "Request"
```

Comment:
Without an action name the request is not displayed properly in matomo.

Reply:
Never mind. I tested this again. Should work fine without the action name

Reply:
Do you plan to remove this then?